### DISCUSSION TITLE
Unattended import with –search-id

### DISCUSSION THREAD

**Discussion by a0g83agbc84** [beetbox/beets#5065] [Q&A] [answered]
Hi everyone,

I want to be able to run completely unattended import tasks. For that, I have individual tracks with the Spotify album ID in their filename, that I can use with import command with `--search-id`.

That works really well as it very quickly matches the track I’m importing with the correct Spotify album. The issue is that even though I am 100% sure that the match is good (as I pass the correct album ID), sometimes beets reports a low enough match so that it still asks for confirmation.

I tried to pass -q but then it skips the import. I tried `quiet_fallback` with only options are `skip` (default one) and `asis`, which completely ignores the album ID match. I, of course, tried `apply` as `quiet_fallback` option but that does not exist.

How can I default to `apply` so that I can have a fully automated import by using the `--search-id` option?

Thanks in advance!

**Comment by arsaboo** [comment]
You can adjust your matching [threshold](https://beets.readthedocs.io/en/latest/reference/config.html#autotagger-matching-options). You will want to increase `strong_rec_thresh` to a number that you feel comfortable with.

**Reply by a0g83agbc84** [reply]
Thanks for your answer. I just tried but it does not seem to work. I tried with very low values but it keeps skipping when running `import` with `-q`. Match is over 70% when running without `-q`.
```
match:
  strong_rec_thresh: 0.20  # auto-apply for matches of 10% or higher
  medium_rec_thresh: 0.20
  rec_gap_thresh: 0.20
```

Note that I tried with only `strong_rec_thresh` but didn't work, so I added the other ones for some extra tests. I can also see in the docs that it says: `You can configure some aspects of the logic beets uses when automatically matching MusicBrainz results under the match: section.`, so it seems to only work with MusicBrainz, but unfortunately I can't get that ID automatically, whereas I can get the Spotify ID automatically.

Any other ideas?

**Reply by arsaboo** [reply]
You want to use large values in your case. What is the matching % that beets shows in the output?

**Reply by a0g83agbc84** [reply]
It does not matter, whether I use 0.01 or 0.99, when using `import` with `-q` it always skips.

Complete query:
`beet import /data/music/downloads/This\ Is\ It.ogg --search-id 5OIWRUGXVlnY5gYP8KCRHV`

 Output:
 ```
 /data/music/downloads/This Is It.ogg (1 items)
Tagging:
    Oh The Larceny - Blood Is Rebel
URL:
    https://open.spotify.com/album/5OIWRUGXVlnY5gYP8KCRHV
(Similarity: 74.3%) (missing tracks, source, tracks) (Spotify, 2019, Position Music)
 * This Is It -> This Is It (source)
Missing tracks (5/6 - 83.3%):
 ! Another Level     (# 1) (3:10)
 ! Real Good Feeling (# 2) (3:14)
 ! Making Moves      (# 3) (3:03)
 ! Turn It Up        (# 4) (3:17)
 ! Stand Up          (# 6) (3:09)
[A]pply, More candidates, Skip, Use as-is, as Tracks, Group albums,
Enter search, enter Id, aBort, eDit, edit Candidates?
```

Output with `-q`:
```
/data/music/downloads/This Is It.ogg (1 items)
Skipping.
```

**Reply by a0g83agbc84** [reply, marked as answer]
@arsaboo I managed to get the behaviour I was looking for with the following configuration:
```
match:
  strong_rec_thresh: 1
  max_rec:
    missing_tracks: strong
    unmatched_tracks: strong
```

With this config, `import` with both `-q` and without it, automatically apply the `--search-id` album.

Thanks for pointing me at the right direction!
